ibm/service/vpc/data_source_ibm_is_lb_profile.go

```diff
@@ -36,9 +36,12 @@ func DataSourceIBMISLbProfile() *schema.Resource {
 							Description: "The type for access mode",
 						},
 						"value": {
-							Type:        schema.TypeString,
+							Type:        schema.TypeList,
```

Comment:
this will be a breaking change, can you see if you can rename the field

Reply:
created new field `values`